packages/user-registry/package.json

```diff
@@ -22,7 +22,7 @@
     "test": "test"
   },
   "scripts": {
-    "build": "yarn build:ts",
+    "build": "yarn build:static && yarn build:ts",
```

Comment:
Nice one. I think we should change `yarn build` in every repo to behave like this.